### PR TITLE
fix: add non-breaking white space to message tags

### PR DESCRIPTION
**Pull request by ryansng**
image

### CODE REVIEW

**Comment by kevmo314** [comment]
No code in the change? But nice haha I was thinking about this yesterday

**Comment by ryansng** [comment]
sorry I forgot to commit :(

File: lib/components/chat_history/twitch/message.dart [hunk 1]
```diff
@@ -113,9 +113,12 @@ class TwitchMessageWidget extends StatelessWidget {
       final userModel = Provider.of<UserModel>(context, listen: false);
       final loginChannel = userModel.userChannel!.displayName;
       if (token.username.toLowerCase() == loginChannel.toLowerCase()) {
-        yield TextSpan(text: " @${token.username} ", style: tagStyleStreamer);
+        yield TextSpan(
```
Comment (kevmo314) [inline lib/components/chat_history/twitch/message.dart:116]: Add a quick comment like `// wrap with nonbreaking spaces` or similar

**Review by kevmo314 — APPROVED** [review]

**Comment by ryansng** [comment]
Twitch's max username length is 25, this is how it looks like on chat history max font size:
image

**Review by kevmo314 — APPROVED** [review]

**Comment by kevmo314** [comment]
Change looks good to me, are you able to merge or is it not letting you? Usually the author merges :)

**Comment by ryansng** [comment]
> Change looks good to me, are you able to merge or is it not letting you? Usually the author merges :)

Yep, I am able to merge. Just wanted your final approval before I resolve conversation and merge!

**Pull request merged**